<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>1348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NAS</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../resources/icons/AppIcon.jpeg</normaloff>../../resources/icons/AppIcon.jpeg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0 y1:0, x2:0 y2:1, stop:0 #373a3c, stop:1 #72777d);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QWidget" name="Header" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>151</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>151</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #373a3c;
border-bottom: 1px solid white;</string>
      </property>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0" columnstretch="0,0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>30</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-bottom: none;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../../resources/icons/FitIcon.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="margin">
          <number>1</number>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font: 63 26pt &quot;Yu Gothic UI Semibold&quot;;
border-bottom: none;</string>
           </property>
           <property name="text">
            <string>Identifikace a verifikace osob s využitím EEG</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 63 12pt &quot;Yu Gothic UI Semibold&quot;;
color: rgb(255, 255, 255);
border-bottom: none;</string>
           </property>
           <property name="text">
            <string>Autor : Roland Žitný - xzitny01</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 63 12pt &quot;Yu Gothic UI Semibold&quot;;
color: rgb(255, 255, 255);
border-bottom: none;</string>
           </property>
           <property name="text">
            <string>Vedúci práce : Ing. Jan Tinka</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="StimuliInfoWidget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <layout class="QVBoxLayout" name="StimuliInfoLayout">
       <property name="topMargin">
        <number>60</number>
       </property>
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="StimPresInfo">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>120</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 12pt Bahnschrift SemiBold;
color: white;</string>
         </property>
         <property name="text">
          <string>Po stlačení tlačidla štart začne zaznamenávanie mozgových signálov.
Ubezpečte sa že máte správne zapojené zaznamenávacie zariadenie a
sústredte sa prosím na zobrazované stimuly.
V prípade nefunkčnosti je potrebné skontrolovať správne nastavanie
portu zariadenia v config.py</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>50</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QPushButton" name="StartRecording">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: qradialgradient(cx:0, cy:0, radius: 1,
                fx:0.5, fy:0.5, stop:0 #81B622 ,stop:0.5 #59981A stop:1 #3D550C);
border-radius: 90px;
border-width: 4px;
border-style: groove;
border-color: #E2DED0;
font: 63 18pt &quot;Bahnschrift SemiBold&quot;;
color: white;
}

QPushButton::hover
{
background-color: qradialgradient(cx:0, cy:0, radius: 1,
                fx:0.5, fy:0.5, stop:0 #91cc27 ,stop:0.5 #5fa31c stop:1 #507010);;
border-color: #adaa9e;
}

QPushButton::pressed {
border-width: 3px;
border-color: #adaa9e;
}</string>
         </property>
         <property name="text">
          <string>Štart</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="StimuliLayoutWidget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <layout class="QVBoxLayout" name="StimuliLayout">
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="StartTimerLabel">
         <property name="styleSheet">
          <string notr="true">font: 80pt &quot;OCR A Extended&quot;;
color: #EEEDE7;</string>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="StimuliImage">
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>600</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>600</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(34, 34, 34);
border: 3px solid black</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
